public/index.html

```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;400&display=swap');
+    </style>
```

Comment:
Hopefully this won't be needed once Meera's PR is up, this snippet will probably be changed once her PR is merged in.

Reply:
<style>
      @import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;400&display=swap');
</style> 
most recent push doesn't have this tag anymore